syncopy/datatype/methods/show.py

```diff
@@ -40,6 +40,10 @@ def show(data, **kwargs):
         determines which keywords can be used.  Some keywords are only valid for
         certain types of Syncopy objects, e.g., "freqs" is not a valid selector
         for an :class:`~syncopy.AnalogData` object.
+    squeeze : bool
```

Comment:
this is great and will hopefully feel quite natural for users (e.g. like numpy behavior)